### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Lgarciac24**
I changed the # character for the word Number

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const data = [
     {
-        title: "Clase #1",
+        title: "Clase Numero 1",
```
Comment (seagomezar) [inline scripts.js:3]: Ah, very bad , resign please!

**Review by seagomezar — CHANGES_REQUESTED** [review]
Super good!

File: scripts.js (hunk 1 above)
Comment (Lgarciac24) [reply scripts.js]: Ok, please review again

File: scripts.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const data = [
     {
-        title: "Clase Numero 1",
+        title: "Clase #1",
```
Comment (seagomezar) [inline scripts.js:3]: Good!

**Review by seagomezar — APPROVED** [review]
Good, please merge!

**Pull request merged**